allofplos/plos_corpus.py

```diff
@@ -385,66 +385,67 @@ def download_amended_articles(directory=None, tempdir=newarticledir, amended_art
+            article.directory = directory
+            if article.proof == 'uncorrected_proof':
+                uncorrected_proofs.add(article.doi)
         print("Saving uncorrected proofs.")
-        with open(uncorrected_proofs_text_list, 'w') as f:
-            for item in tqdm(sorted(uncorrected_proofs_list), disable=None):
+        with open(list_path, 'w') as f:
+            for item in tqdm(sorted(uncorrected_proofs), disable=None):
                 f.write("%s\n" % item)
-    return uncorrected_proofs_list
+    return uncorrected_proofs
 
 
-def check_for_uncorrected_proofs(directory=newarticledir, text_list=uncorrected_proofs_text_list):
+def check_for_uncorrected_proofs(directory=newarticledir, list_path=uncorrected_proofs_text_list):
     """
     For a list of articles, check whether they are the 'uncorrected proof' type
     One of the checks on newly downloaded articles.
-    :param text_list: List of DOIs
+    :param list_path: List of DOIs
     :param directory: Directory containing the article files
-    :return: all articles that are uncorrected proofs, including from main article directory
+    :return: set of all articles that are uncorrected proofs, including from main article directory
     """
 
     # Read in uncorrected proofs from uncorrected_proofs_text_list txt file
-    # If uncorrected_proofs_list txt file doesn't exist, build that list from scratch from main article directory
-    uncorrected_proofs_list = get_uncorrected_proofs_list()
+    # If uncorrected_proofs txt file doesn't exist, build that set from scratch from main article directory
+    uncorrected_proofs = get_uncorrected_proofs(list_path=list_path)
 
     # Check directory for uncorrected proofs
-    # Append uncorrected proofs to running list
+    # Append uncorrected proofs to running set
     articles = listdir_nohidden(directory)
     new_proofs = 0
     for article_file in articles:
         article = Article.from_filename(article_file)
         article.directory = directory
         if article.proof == 'uncorrected_proof':
-            uncorrected_proofs_list.append(article.doi)
+            uncorrected_proofs.add(article.doi)
             new_proofs += 1
     # Copy all uncorrected proofs from list to clean text file
-    with open(text_list, 'w') as f:
-        for item in sorted(set(uncorrected_proofs_list)):
+    with open(list_path, 'w') as f:
+        for item in sorted(uncorrected_proofs):
             f.write("%s\n" % item)
-    if uncorrected_proofs_list:
-        print("{} new uncorrected proofs found. {} total in list.".format(new_proofs, len(uncorrected_proofs_list)))
+    if uncorrected_proofs:
+        print("{} new uncorrected proofs found. {} total in set.".format(new_proofs, len(uncorrected_proofs)))
     else:
-        print("No uncorrected proofs found in folder or in existing list.")
-    return uncorrected_proofs_list
+        print("No uncorrected proofs found in folder or in existing file.")
```

Comment:
instead of saying folder and file, maybe state the directory name and the file name (using `"".format()`)?